app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -282,6 +313,16 @@ class WireActivityViewModel @Inject constructor(
         }
     }
 
+    fun dismissNewClientDialog() {
+        globalAppState = globalAppState.copy(newClientDialog = null)
+    }
+
+    fun switchAccount(userId: UserId) {
```

Comment:
Would that be possible to switch account and open the `SelfDevices` for that switched account? Maybe by using deep links just like you are implementing right now for conversations and calls? That would be way better UX I think 😄

Reply:
yeah, from now we can achieve it easily :D